edit-creature.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+################################################################################
+# This script toggles the 'tool' keyword for our creature scripts as a
+# workaround for Godot #40000
+# (https://github.com/godotengine/godot/issues/40000)
+#
+# edit-creature.sh on: Enable creature editing in the editor. This causes errors
+#     in the Godot console.
+#
+# edit-creature.sh off: Disable creature editing in the editor. This fixes the
+#     errors in the Godot console.
+
+tool_line='tool #uncomment to view creature in editor'
+
+if [ "$1" = "on" ]
+then
+  # replace 'tool #uncomment' with '#tool #uncomment'
```

Comment:
This is hard to read for humans. Maybe something more like `# comment out tool lines`

Reply:
Rephrased to `# uncomment 'tool' lines to allow editing`